index.js

```diff
@@ -93,7 +87,7 @@ app.get('/categorias/:category', async function (request, response) {
 app.post('/api/checkout', async function (request, response) {
     try {
         const { id, amount } = request.body;
-        const payment = await stripe.paymentIntents.create({
```

Comment:
You do not use the payment const, consider re adding in case it's needed and leave a comment explaining it.